app/assets/javascripts/app/views/header_view.js

```diff
@@ -12,12 +12,18 @@ app.views.Header = app.views.Base.extend({
     });
   },
 
-  postRenderTemplate: function(){
-    new app.views.Notifications({ el: "#notification-dropdown" });
-    this.notificationDropdown = new app.views.NotificationDropdown({ el: "#notification-dropdown" });
-    new app.views.Search({ el: "#header-search-form" });
+  postRenderTemplate: function() {
+    if (!app.collections.Notifications.instance) {
+      app.collections.Notifications.instance = new app.collections.Notifications();
+    }
```

Comment:
I'd like an advice on this. The collection needs to be shared with [the notification view of the `/notifications` page](https://github.com/diaspora/diaspora/pull/6952/files#diff-829437a96c9bf5dc88c42fa29c90e469R137) so I create a class instance but I'm not really satisfied with this solution.